src/theory/quantifiers/sygus/sygus_unif_io.cpp

```diff
@@ -560,7 +561,7 @@ void SygusUnifIo::notifyEnumeration(Node e, Node v, std::vector<Node>& lemmas)
   Assert(!d_examples.empty());
   Assert(d_examples.size() == d_examples_out.size());
 
-  EnumInfo& ei = d_strategy[c].getEnumInfo(e);
+  EnumInfo& ei = d_strategy.at(c).getEnumInfo(e);
```

Comment:
Whats the motivation for this change?

Reply:
You can't use `[c]` because `SygusUnifStrategy` doesn't have a default constructor -- it takes `Env`. So you have to make sure that it is already inserted, you can't default insert if the an entry with they key `c` doesn't exist. From what I understand from the code, that's actually the invariant here anyways, so it would be better practice to always use `at` if we assert that it exists and no new entry is to be inserted. It would better document what is going on here (for cases where there is a default constructor).

Reply:
Ah, I see, make sense.